### PR TITLE
Improve installation docs

### PR DESCRIPTION
**Pull request by nickolay**

### CODE REVIEW

File: docs/source/installation.rst [hunk 1]
```diff
@@ -50,8 +50,8 @@ the first line is required::
     DATABASES = {
         'default': {
             'ENGINE': 'django.db.backends.mysql',
-            'NAME': '',
-            'USER': '',
+            'NAME': 'flightdeck',
+            'USER': 'root',
```
Comment (seanmonstar) [inline docs/source/installation.rst:54]: the user may be another user, i think it's sufficiently clear leaving it blank.

File: docs/source/installation.rst [hunk 2]
```diff
@@ -74,29 +74,65 @@ the first line is required::
     CACHES['default']['BACKEND'] =
     'django.core.cache.backends.locmem.LocMemCache'
 
+Make sure that MySQL is running, then create the database you specified
+in settings_local.py::
+
+    mysql -u root -p
```
Comment (seanmonstar) [inline docs/source/installation.rst:80]: here as well, the user might be something other than root, and not necessarily require a password.

**Comment by seanmonstar** [comment]
Other than the 2 references to a root, this is awesome! Thanks!

File: docs/source/installation.rst (hunk 1 above)
Comment (nickolay) [inline docs/source/installation.rst:54]: There must be a user though. In the case the developer is familiar with MySQL and has it installed and configured, he'll know he can change the user. (You could add a note about that if you like.)

For a developer who doesn't care about his MySQL configuration, just seeing it as an obstacle to getting Flightdeck running, having a pre-filled template is very helpful.

**Comment by seanmonstar** [comment]
Very well, I see little harm.

Again, much thanks!

**Comment by nickolay** [comment]
Thanks for quick turnaround!

**Pull request merged**